src/neo/models/neo4j/actor.js

```diff
@@ -0,0 +1,10 @@
+const _ = require('lodash');
```

Comment:
I expect these models to be extended with things like date parsing as soon as the migration is ready.